operations/services.md

```diff
@@ -281,7 +281,7 @@ Returns selected availability and occupancy metrics of a bookable service for a
 
```

Comment:
FYI: not directly related to this PR, just some random code change that changed the documentation